app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "If you're disabled"
+heading: "If you're disabled"
+description: |-
+    Find out what extra support and adjustments you could be entitled to when you're training to teach if you're disabled.
+related_content:
+    Who do you want to teach?: "/train-to-be-a-teacher/who-do-you-want-to-teach"
+    Becoming a teacher with a hearing impairment: "/blog/overcoming-challenges-to-become-a-teacher-hearing-impairment"
+    Becoming an autistic teacher: "/blog/overcoming-challenges-to-become-a-teacher-autism"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-support-promo
+navigation: 20.20
+navigation_description: Find out what extra support and adjustments you could be entitled to when you're training to teach if you're disabled.
+keywords:
+    - Disability
+    - Disabilities
+    - DSA
+    - Financial Support
+    - Disabled students
+    - Disabled Students’ Allowances
+---
+
+The experience and perspective of a diverse workforce is valued in school culture.
+
+If you’re disabled, have a mental health condition or educational needs, you can get support to become a teacher.
+
+## Disabled Students’ Allowance and other support
+
+Disabled Students’ Allowance (DSA) is support to cover the study-related costs you have because of a mental health problem, long term illness or any other disability
+
+[Find out more about DSA](https://www.gov.uk/disabled-students-allowance-dsa).
+
+[If you do paid training you may also be eligible for support](https://www.gov.uk/access-to-work).
+
+[AbilityNet](https://abilitynet.org.uk/about-abilitynet) or [Disability Rights UK](https://www.disabilityrightsuk.org/adjustments-disabled-students) may also be able to help you identify technology or other adjustments to help you study.
+
+## Adjustments to help you train
+
+Teacher training providers can make adjustments so that you can attend an interview or do the training. For example, they could:
+
+- organise equipment like a hearing loop or an adapted keyboard
+- put you in touch with support staff if you have a mental health condition
+- make sure classrooms are wheelchair accessible
+
```

Comment:
We should encourage candidates to contact providers to feel reassured at this stage,